### PR TITLE
override logger path

### PR DESCRIPTION
**Pull request by rupesh9**

### CODE REVIEW

File: lib/api_call_cache.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def self.api_cache_redis_instance(redis_inst)
   end
 
   def self.logger_path(log_file = 'log/api_call_cache.log', rotation = 'monthly')
-    @@acc_logger ||=  Logger.new(log_file, rotation)
+    @@acc_logger =  Logger.new(log_file, rotation)
```
Comment (akshayrao14) [inline lib/api_call_cache.rb:23]: @rupesh9 why was this required?

Comment (rupesh9) [reply lib/api_call_cache.rb:23]: @akshayrao14 Logger is initialize with nil logging path and when we pass the path at app initialization @@acc_logger is not nil so the logging is not done.
